src/server/scripts/Spells/spell_item.cpp

```diff
@@ -1501,6 +1501,39 @@ class spell_item_arcane_shroud : public SpellScriptLoader
         }
 };
 
+// 64415 - Val'anyr Hammer of Ancient Kings - Equip Effect
+class spell_item_valanyr_hammer_of_acient_kings : public SpellScriptLoader
```

Comment:
typo: its ancient

Reply:
Thanks for pointing that out 👍  fixed the typo